### PR TITLE
Velvet tutorial updates

### PR DESCRIPTION
**Pull request by AnnaSyme**
Some minor changes.

### CODE REVIEW

**Review by hexylena — DISMISSED** [review]

File: topics/assembly/tutorials/general-introduction/tutorial.md [hunk 1]
```diff
@@ -124,22 +124,25 @@ We will evaluate the input reads using the FastQC tool. This tool runs a standar
 >    {% snippet faqs/galaxy/tools_select_multiple_datasets.md %}
 >  
 > 2. **MultiQC** {% icon tool %} with the following parameters
```
Comment (hexylena) [inline topics/assembly/tutorials/general-introduction/tutorial.md:126]: @AnnaSyme could I trouble you to add the current versions of these tools to annotate this tutorial?

```suggestion
> 2. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.9+galaxy1) %} with the following parameters
```

but please check against which versions you used of course

Comment (AnnaSyme) [reply topics/assembly/tutorials/general-introduction/tutorial.md:126]: Thanks @hexylena - yes I will do this

Comment (AnnaSyme) [reply topics/assembly/tutorials/general-introduction/tutorial.md]: have added to the pull request - not always the latest tools in this case as it's to match a particular training session- hope that's ok!

**Review by hexylena — APPROVED** [review]

File: topics/assembly/tutorials/general-introduction/tutorial.md (hunk 1 above)
Comment (hexylena) [reply topics/assembly/tutorials/general-introduction/tutorial.md]: No, that's ideal! Knowing what it is tested with + works is more useful than just writing the latest versions

**Comment by hexylena** [comment]
grouse, thanks @AnnaSyme !

**Pull request merged**